### PR TITLE
done the task to update the title of a existing poll in all poll list

### PR DESCRIPTION
**Pull request by ravi1225**

### CODE REVIEW

File: src/components/PollData.js [hunk 1]
```diff
@@ -1,21 +1,41 @@
...
+
+
+    let temp = "";
+    let temp1 = "";
+
+    if (editTitle.length > 0) {
+        const { title, _id } = editTitle[0]
+        temp = title,
+            temp1 = _id
+    }
```
Comment (priyarajputexc) [inline src/components/PollData.js:37]: what is the use of this block?

Comment (ravi1225) [reply src/components/PollData.js:37]: With the help of this we can get the existing title and id of the poll

Comment (priyarajputexc) [inline src/components/PollData.js:37]: what is the use of this block?

File: src/components/PollData.js [hunk 2]
```diff
@@ -30,27 +42,56 @@ const PollData = (props) => {
             { cancelable: false }
         );
 
+    const updatePollTitle = (title, id) => {
+        showModal(true)
+        const arr = props.pollData.filter((e) => e._id == id)
```
Comment (priyarajputexc) [inline src/components/PollData.js:47]: use find instead of filter.
then no need access arr[0]
it will return the object

**Pull request merged**